binder-index.md

```diff
@@ -75,6 +75,14 @@ These are noted in the README.md files for each sample, along with complete inst
       <td>Q# standalone</td>
       <td></td>
     </tr>
+    <tr>
+      <td></td>
+      <td><strong><a href="./samples/getting-started/azure-quantum/README.md">Qiskit and Cirq</a></strong></td>
+      <td><a href="./samples/getting-started/azure-quantum">Jupyter notebooks</a></td>
+      <td></td>
+      <td>N/A</td>
```

Comment:
I'm noticing from this as well that the path is `samples/getting-started/azure-quantum`, which would imply that these notebooks are _the_ way to get started with services? I'd suggest either under `samples/azure-quantum` or `samples/interoperability` would better communicate that this is one of many different ways to work with our services. As a future improvement, we may want to also separate this into multiple folders, following the way other samples are laid out; that will be especially helpful when publishing from the samples repo to other ways of reading samples (e.g.: the Samples Browser).

With respect to the binder index itself, the first column after the sample name has generally been used for quantum programs directly written in a Jupyter Notebook (e.g.: "Q# notebook") rather than programs generated by or called from a host language (e.g.: "Q# + Python"). Following that pattern, using the third column to indicate something like "Cirq/Qiskit + Python" would likely fit with other samples better:

```suggestion
      <td><strong><a href="./samples/getting-started/azure-quantum/README.md">Qiskit and Cirq</a></strong></td>
      <td></td>
      <td><a href="./samples/getting-started/azure-quantum">Cirq/Qiskit + Python</a></td>
      <td>N/A</td>
```